ehitchhiking-ui/src/app/shared/components/active-trip/active-trip.component.sass

```diff
@@ -0,0 +1,56 @@
+@import '../../../../styles/fonts-constants'
+@import '../../../../styles/colors'
+@import '../../../../styles/constants'
+@import '../../../../styles/mixins'
+
+.info
```

Comment:
pls remove extra styles

Reply:
common styles for active trip-component and trip-component move to separate file

Reply:
created file trip.sass, removed extra styles